nixos/modules/services/misc/home-assistant.nix

```diff
@@ -107,7 +118,9 @@ in {
       after = [ "network.target" ];
       preStart = lib.optionalString (cfg.config != null) ''
         rm -f ${cfg.configDir}/configuration.yaml
-        ln -s ${configFile} ${cfg.configDir}/configuration.yaml
+        ${pkgs.remarshal}/bin/json2yaml \
+          -i ${configFile} \
+          -o ${cfg.configDir}/configuration.yaml
```

Comment:
This file can be written to by Home Assistant if it isn't a symbolic link. We'll probably want to `chmod` it.

Reply:
I thought a bit about that because as HA is adding additional configuration options to the web interface, it's likely that people would want to make some changes that way and then migrate it over to ```config```. If HA can write to the file, you can than diff it later to see what changed but maybe we just wait until that becomes an issue. Let's not change behaviour now - so I set it to ```400```.